app/human-detector-app/__tests__/BackendService.test.ts

```diff
@@ -25,12 +25,12 @@ describe(BackendService, () => {
   // test putting notification key
   describe('sendNotifyTokenAPI', () => {
     const backendService = new BackendService(mockTokenManager);
-    const newUser: User = new User('tucker', '987654', true);
+    const newUser: User = new User('tucker', '987654', []);
 
     it('should return code 200 for successful token sending', async () => {
       mockTokenManager.getUser.mockReturnValue(newUser);
       nock(ServerConfig.apiLink)
-        .put(ServerConfig.getSendNotifKeyUrlExtension(newUser.userID))
+        .put(ServerConfig.getSendNotifKeyUrlExtension(newUser.getUserId))
```

Comment:
```suggestion
        .put(ServerConfig.getSendNotifKeyUrlExtension(newUser.userId))
```
Same for the rest of the instances of this in this file